docs/src/layout/containerDeprecated/containerDeprecated.jsx

```diff
@@ -0,0 +1,48 @@
+import {
```

Comment:
Are we planning on changing `<Container>` to `<ContainerDeprecated>`?
This deprecation I think I at least expected ( `<Header>` and `<Dropdown>` caught me a little bit off-guard).

Reply:
I don't think we should rename these in `./src`, but in `./docs/src` my thinking was/is to better identify what is deprecated. I could be wrong. Let me know.